### ISSUE TITLE
[BUG]: Expanding and loading state renders cell with undefined values

### ISSUE THREAD

**Issue by nichita-pasecinic** [KevinVandy/material-react-table#713]
### material-react-table version

1.15.0

### react & react-dom versions

17.0.2

### Describe the bug and the steps to reproduce it

When loading is true Cell component gets rendered and `cell.getValue()` is undefined

```ts
<MaterialReactTable
      columns={[
        {
          id: 'someId',
          accessorFn: (x) => x.firstName,
          header: 'First Name',
          Cell: ({ cell }) => {
            console.log('cell value: ', cell.getValue()); // <----- here cell.getValue() is always undefined when loading is true
            // the Cell component gets rendered;
            return <>yo</>;
          },
        },
      ]}
      state={{ isLoading: true }}
      data={[]}
      enableExpanding
    />
```

### Minimal, Reproducible Example - (Optional, but Recommended)

https://stackblitz.com/edit/github-wtvpnu-zgsit2?file=src%2FTS.tsx,package.json,src%2FmakeData.ts

### Do you intend to try to help solve this bug with your own PR?

None

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by nichita-pasecinic** [comment]
@KevinVandy this PR introduced it - https://github.com/KevinVandy/material-react-table/pull/601

**Comment by Doccrazy** [comment]
Related problem: when isLoading is true, the contents of the action column are still rendered. This was not the case in 1.14.0.

See sandbox for example: https://stackblitz.com/edit/github-yb9axs?file=src%2FTS.tsx

**Comment by AidenHadisi** [comment]
Facing the same issue here. downgrading to 1.14.0 solves the issue.

**Comment by KevinVandy** [comment]
yeah, seems like this and #721 are because of this change. Going to be reverting this behavior

**Comment by KevinVandy** [comment]
fixed in v2, backported to v1.15.1
